Tools/nib2cib/NSBox.j

```diff
@@ -71,9 +71,8 @@
         }
 
         _title         = [[aCoder decodeObjectForKey:@"NSTitleCell"] objectValue] || @"";
-        _titlePosition = [aCoder decodeObjectForKey:@"NSTitlePosition"];
 
-        if (_titlePosition === undefined)
+        if (_titlePosition === undefined && _boxType !== CPBoxSeparator)
```

Comment:
@cacaodev If you look at the history of this line in the git repository there is some kind of fix for when the ```_titlePosition``` is not present. I suggested this fix as Xcode now has removed the title position from the .xib file for line separators, starting from I think Xcode 8. Maybe it is better to check if the attribute is missing with the ```containsValueForKey``` method but I think it is not in the scope of this pull request as @enquora says. This pull request is just to fix the new behaviour of Xcode.